### DISCUSSION TITLE
How to operate on TIME record with prisma using MySQL?

### DISCUSSION THREAD

**Discussion by akandil7** [prisma/prisma#7192] [Q&A] [answered]
prisma schema: ```DateTime @db.Time(0)```

**How to delete a record having a certain time?**

Knowing that while inserting the record, it had a value yyyy-mm-dd'T'hh:mm:ss.SSSZ 
But then couldn't delete the record using the same yyyy-mm-dd'T'hh:mm:ss.SSSZ  because the date part was changed to : 1970-01-01?

Thank you in advance

**Comment by ryands17** [comment, marked as answer]
@akandil7 👋 

The `time` type only stores the time part so you would need to construct the date object with the required time you want to query and the date part set to **1970-01-01**.

**Reply by akandil7** [reply]
@ryands17 Thank you very much for your help
